### ISSUE TITLE
Too Many rows?

### ISSUE THREAD

**Issue by mtzirkel** [jjuback/gc-excelviewer#143]
Hello. Great extension. I have 4 .xlsx sheeting in my project. three of them with rows of about 500 will Preview. The one with 15,000 rows will not. Is this some wort of max rows limit? All files will open in libreCalc. Am I missing something like pandas head() to just get the first few rows?

Thanks Great extension

**Comment by abdullzz** [comment]
yeah i also have same problem with 400000 rows. seems like this extension cannot handle it well

**Comment by JBuursma** [comment]
Copying this relevant discussion from the Rating & Review section of the extension marketplace page:
- Dmitry Demidov: "50Mb file size limit"
- GrapeCity: "This is a documented limitation of VS Code itself, and there is nothing any extension can do to override it."

Suggestion: show what you can (do an automatic head). Ideally, notify the user that you're showing a truncated number of rows, even if it's just including the word "head" in the tab title. Is this possible?

(In general, thank you for your extension!)
